src/components/common/approval/ApproveToken.tsx

```diff
@@ -221,7 +221,9 @@ export const ApproveToken: FC<Props> = ({
           </span>
         )}
 
-        {error ? <pre>{JSON.stringify(error, null, 2)}</pre> : null}
+        {error ? (
+          <pre className="max-w-min text-wrap">{error.message}</pre>
```

Comment:
Maybe use `output` if it's based on an form result.

Reply:
Agree